src/Http/Controllers/AdminController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers;
+
+use Blumilk\Meetup\Core\Models\Meetup;
+use Blumilk\Meetup\Core\Models\News;
+use Blumilk\Meetup\Core\Models\Organization;
+use Blumilk\Meetup\Core\Models\Speaker;
+use Blumilk\Meetup\Core\Models\User;
+use Illuminate\Contracts\View\View;
+
+class AdminController extends Controller
+{
+    public function dashboard(): View
+    {
+        return view("admin.dashboard");
+    }
+
+    public function usersIndex(): View
+    {
+        $users = User::query()->latest()->paginate(20);
```

Comment:
Please avoid to use magic numbers like this `20` here. Assign a value to some constans. It can be in this particalar controller, in base controller for all paginated stuff, in some config or in some helper.

Reply:
Config file where we can edit parameters of our app will be okay.